.github/workflows/rpc-tests.yml

```diff
@@ -24,33 +24,43 @@ jobs:
           # Apple Intel (this runner has better support to nested virtualization)
           - macos-15-large
         exclude:
-          # Only run Linux x64 tests on pull request to save some time
-          - sys:
-              ${{ github.event_name != 'push' && 'ubuntu-jammy-8-cores-arm64' }}
-          - sys: ${{ github.event_name != 'push' && 'macos-15-large' }}
+          # Only run Linux x64 on non-release PRs to save CI minutes
+          - sys: ${{ github.event_name != 'push' && !startsWith(github.ref_name, 'release/') && 'ubuntu-jammy-8-cores-arm64' }}
+          - sys: ${{ github.event_name != 'push' && !startsWith(github.ref_name, 'release/') && 'macos-15-large' }}
```

Comment:
Thoughts on only running all 3 container types on release branches? It might not be necessary for main pushes / feature PRs